adding final dropdowns for feature flag dev hub menu (#7156)
harness/developer-hub: docs/feature-flags/use-ff/ff-creating-flag/using-ff-ccm.md

#### Before you begin

> * The same connector cannot be used in NextGen and FirstGen. For information on creating a GCP connector in the FirstGen see [Set Up Cost Visibility for GCP](../../../first-gen/cloud-cost-management/setup-cost-visibility/enable-cloud-efficiency-for-google-cloud-platform-gcp.md).

Reviewer: ```suggestion
* You can't use the same connector in NextGen and FirstGen. For information on creating a GCP connector in the FirstGen, go to [Set Up Cost Visibility for GCP](../../../first-gen/cloud-cost-management/setup-cost-visibility/enable-cloud-efficiency-for-google-cloud-platform-gcp.md).
```
